contracts/vaults/Claim.sol

```diff
@@ -4,7 +4,7 @@ pragma solidity 0.8.6;
 import "./Base.sol";
 
 contract Claim is Base {
-    using SafeERC20 for IERC20;
+    using SafeERC20Upgradeable for IERC20Upgradeable;
```

Comment:
why? we are calling that token, we do not care if it upgradeable or not, or do we?